### ISSUE TITLE
Landing page for docs violates a11y animation requirements

### ISSUE THREAD

**Issue by HarelM** [themustafaomar/libregl#1]
There is an accessibility issue in the main page of the docs - there's a a11y standard to make sure animation doesn't last more than 5 seconds or it can be paused by a button.

We had a similar animation in maplibre.org but were told we should remove it.
Just FYI :-)

**Comment by themustafaomar** [comment]
Thanks for letting me know about that, gonna getting rid of the background animation as soon as possible.
